sechub-administration/src/test/java/com/mercedesbenz/sechub/domain/administration/user/AnonymousUserRestControllerTest.java

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.domain.administration.user;
+
+import static com.mercedesbenz.sechub.test.RestDocPathParameter.ONE_TIME_TOKEN;
+import static com.mercedesbenz.sechub.test.SecHubTestURLBuilder.https;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.security.test.context.support.WithAnonymousUser;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import com.mercedesbenz.sechub.domain.administration.TestAdministrationSecurityConfiguration;
+import com.mercedesbenz.sechub.sharedkernel.Profiles;
+import com.mercedesbenz.sechub.spring.security.SecHubSecurityProperties;
+import com.mercedesbenz.sechub.test.TestPortProvider;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest
+@ContextConfiguration(classes = { AnonymousUserRestController.class })
+@ActiveProfiles({ Profiles.TEST })
+@Import(TestAdministrationSecurityConfiguration.class)
+public class AnonymousUserRestControllerTest {
+
+    private static final int PORT_USED = TestPortProvider.DEFAULT_INSTANCE.getWebMVCTestHTTPSPort();
```

Comment:
Hm... normally we have integration tests here which do test this.
But I see this is faster and we test here that no content is returned when
...

and that is unclear: 
userEmailAddressUpdateService is injected as mock, okay, but what kind of behavior does lead to the situation that no content is delivered back here? The kind of behavior is not mentioned here - I would expect something like


when(userEmailAddressUpdateService.xyz).thenReturn(...); // we simulate service behaves....

Also a verify(userEmailAddressUpdateService).xyz would be nice.